controllers/handlers/kubevirt.go

```diff
@@ -853,6 +854,9 @@ func getFeatureGateChecks(featureGates *hcov1beta1.HyperConvergedFeatureGates, a
 	if ptr.Deref(featureGates.AlignCPUs, false) {
 		fgs = append(fgs, kvAlignCPUs)
 	}
+	if featureGates.VideoConfig != nil && *featureGates.VideoConfig {
```

Comment:
Please keep the consistency of the code, e.g.
 
```suggestion
	if ptr.Deref(featureGates.VideoConfig, false) {
```

Reply:
Thank you, fixed